CHANGELOG.md

```diff
@@ -2,8 +2,11 @@
 
 ## [v-master](https://github.com/dallinger/dallinger/tree/master) (xxxx-xx-xx)
 
+## [v-6.1.0](https://github.com/Dallinger/Dallinger/tree/v6.0.0) (2020-04-10)
 - No longer retry `/launch` route in debug mode. Additional logging for launch retries.
 - Allow setting of separate optional `dyno_type_web` and `dyno_type_worker` parameters.
+- Regression fix: experiment files with non-ascii characters in file names are again supported
```

Comment:
More important is that apostrophes are supported again.

Reply:
Good point; I'll fix.